test/unit_tests/user/RichProfileRepositorySpec.js

```diff
@@ -35,7 +35,7 @@ describe('RichProfileRepository', () => {
 
       return richProfileRepository.getUserRichProfile(userId).then(richProfile => {
         expect(richProfileRepository.backendClient.sendRequest).toHaveBeenCalledTimes(1);
-        expect(richProfile).toBe(richProfile);
```

Comment:
Why? This test has lost value with this change.

Reply:
Maybe `toEqual` is the one you are looking for (if the problem is that it's a new object)